### DISCUSSION TITLE
mpv vs. QuickTime on macMini

### DISCUSSION THREAD

**Discussion by morganLJ** [mpv-player/mpv#15406] [General]
Hello

Completely new to mpv - I need it to play a 4K video (5.1 surround) on a macMini.
Mac is OS 10.11.3 (cannot OS-update since it's not mine - long story)
Video is ProRes4444 (163GB for about an hour)

I have been struggling to even install mpv (tried different versions from laboratory.stolendata)

But my main question here: if QuickTime renders the video quite flawlessly to a surround-sound-system and a projector.
And mpv lags and drops audio almost instantly.
Is this something I can do something about? Or just a fact - perhaps due to the ProRes codec?

(When running QuickTime I am using about 60% of the CPU.)

**Comment by Akemi** [comment]
that's a hard to answer question. for one, 10.11 is not supported by mpv anymore. if you are using cocoa-cb/opengl rendering might be the actual bottleneck. quicktime on the other hand is way more optimised int hat regard. hwdec might help in that case, if it even supports decoding prores.

one would first need to find the underlying problem for a definite answer,
